docs/UserGuide.md

```diff
@@ -62,61 +62,105 @@ TimesTable is a **desktop app for managing your tuition students and classes, op
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* For commands that require next-ok-kin (NOK) information, all inputs regarding NOK should be after `nok/` and all 
+  inputs regarding the student should be before `nok/`. Timestable does not allow order swap!<br>
+  e.g. `n/John p/97833242 … /nok n/Harry p/87738383 …` John and 97833242 are information belonging to the student
+  and John's NOK's name is Harry whose phone number is 87738383.
+
+
 </div>
 
 ### Viewing help : `help`
 
+
 Shows a message explaning how to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 
-### Adding a student: `add`
+
+## Adding a student: `add`
+<hr>
 
 Adds a student to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: 
+```
+add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS ct/CLASS_TIMING l/LOCATION r/HOURLY_RATE [t/TAG]… nok/ n/NOK_NAME 
+    p/NOK_PHONE_NUMBER e/NOK_EMAIL   
+```
+
+* This is a command that requires next-of-kin (NOK) information.
+* This command is split into two segments (excluding command keyword). The first segment are the inputs before
+`nok/` and the second segment are the inputs after `nok/`. 
+  * The segments are not fixed in order and inputs in the 
+  first segment are about student information whereas inputs in the second segment are about NOK's information.
+* The order of input within its own segment is swappable. 
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A student can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+ 
+```
+add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 ct/Mon 2-4pm 
+    l/Hougang ave 5 Block 614 #11-419 r/$50 t/ALevels nok/ n/Mary Doe p/93334848 e/mary23@gmail.com
+```
+
+```
+add n/Betsy Crowe ct/Mon 5-7pm l/Serangoon Nex  t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 
+    t/slow learner r/$70 nok/ n/Karen e/karenSUper@gmail.com p/99994444
+```
 
-### Listing all students : `list`
+
+## Listing all students : `list`
+<hr>
 
 Shows a list of all students in the address book.
 
 Format: `list`
 
-### Editing a student : `edit`
+## Editing a student : `edit`
+<hr>
 
 Edits an existing student in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [ct/CLASS_TIMING] [r/HOURLY_RATE] [t/TAG] [l/LOCATION]…​`
+Format: 
+```
+edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [ct/CLASS_TIMING] [r/HOURLY_RATE] [t/TAG] [l/LOCATION]…​
+[nok/] [p/NOK_NAME] [p/NOK_PHONE_NUMBER] [e/NOK_EMAIL]
```

Comment:
would putting location beside class timing be better? or it doesn't matter for the edit format

Reply:
has been updated